src/main.rs

```diff
@@ -169,6 +169,7 @@ fn main() {
             format!("vector={}", level),
             format!("codec={}", level),
             format!("file_source={}", level),
+            format!("tower_limit=trace"),
```

Comment:
actually, do we want to use `{}` here?

Reply:
There's only one trace event in the whole crate and it's at `trace` level, so it's the only level that will ever show anything.

I'd actually like for this to be at `warn`, but that seemed aggressive to hardcode into a library. If there were a way to translate into our desired levels, then I'd definitely use `{}` here.

Reply:
Well this might happen more often this will enable it for every log level. That seems a bit wrong?

Reply:
The alternative would be that it's only ever logged when we're at `trace` everywhere, which I don't think fixes the issue of these being visible during normal operation.

Reply:
fair point, not sure how to solve that then :(

Reply:
Have people thought much about appropriate log levels for libraries? We could change tower-limit to `warn` here, but I'm not sure that's something everyone would want.

Reply:
@hawkw any thoughts?